### DISCUSSION TITLE
Endpoint /livewire/update can be hit outside of laravel application

### DISCUSSION THREAD

**Discussion by abdulrehman25** [livewire/livewire#8374] [7. Bugs (archived)]
### Livewire version

v3.2

### Laravel version

v10

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Any Livewire Update request can be copied from network tab and imported in postman and it will work simply as working in the browser, which is basically violating the usage of the checksum and livewire _token included in the body of the update request.

### Repository or Wirebox URL

---

### Additional information

I have added middleware checks and other checks inside the component but it still works with the _token and checksum from the browser call.

I think there should be randomized checksum which will prevent such issue, like the csrf token in laravel.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [ ] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by francoism90** [comment]
Do you mean the tokens generated by Livewire itself? Do you have an example where this can lead to issues?

I think it works in Postman, because the tokens are valid.
